src/components/Pricing/PricingCard.jsx

```diff
@@ -0,0 +1,178 @@
+import { Button } from '../Common/Button'
+import clsx from 'clsx'
+import NumberCount from './NumberCount'
+import usePricing from './usePricing'
+
+function CheckIcon({ className }) {
```

Comment:
We have 2 icon libraries in the package json, does none of them has a check icon?

Reply:
Replaced with heroIcon.